### ISSUE TITLE
Reproducibility issue?

### ISSUE THREAD

**Issue by aashikrasool** [saturnian77/ASID#6]
I  have tried to reproduce your results while training 500 epochs(learning rate of 5e-4) and finetuned 1000 epochs (learning rate of 4e-4), but it's showing significant drops around 2-3psnr in every scale factor for all datasets. Is there any suggestion or any specific method or loss you used for the fine-tuning phase to get your reported results?
